assets/upgradePatches.json

```diff
@@ -135,6 +135,18 @@
         "name": "kubevirt-plugin-service",
         "namespace": "kubevirt-hyperconverged"
       }
+    },
+    {
+      "semverRange": "<1.10.0",
+      "groupVersionKind": {
+        "group": "tektontasks.kubevirt.io",
```

Comment:
that should be empty as well, this is the API group of CRD itself.

Reply:
Done.